zou/utils/movie.py

```diff
@@ -118,10 +118,32 @@ def get_all_frames(movie_path):
     return folder_path
 
 
-def generate_tile(movie_path):
+def generate_tile(movie_path, movie_fps):
     """
-    ffmpeg -i {movie_path} -vf 'scale=150:100,tile=8x8' -an -vsync 0 tile%03d.png
+    Generates a tile from a movie.
     """
+    folder_path = os.path.dirname(movie_path)
+    file_source_name = os.path.basename(movie_path)
+    file_target_name = f"{file_source_name[:-4]}_tile.png"
+    file_target_path = os.path.join(folder_path, file_target_name)
+    command = [
```

Comment:
Please use the ffmpeg wrapper (https://pypi.org/project/ffmpeg-python) to do that. 
https://stackoverflow.com/questions/31024968/using-ffmpeg-to-obtain-video-durations-in-python